handsontable/src/plugins/formulas/__tests__/featureIntegration.spec.js

```diff
@@ -36,6 +36,73 @@ describe('Formulas: Integration with other features', () => {
 
       expect(hot.countCols()).toEqual(5);
     });
+
+    it('should work properly when indexes are reorganised and some rows/columns are inserted', () => {
+      handsontable({
+        data: [
+          [1, '=A1+10', '=B1+100', '=C1+1000', '=D1+1000000'],
+          [2, '=A2+10', '=B2+100', '=C2+1000', '=D2+1000000'],
+          [3, '=A3+10', '=B3+100', '=C3+1000', '=D3+1000000'],
+          [4, '=A4+10', '=B4+100', '=C4+1000', '=D4+1000000'],
+          [5, '=A5+10', '=B5+100', '=C5+1000', '=D5+1000000'],
+        ],
+        formulas: {
+          engine: HyperFormula,
+          sheetName: 'Sheet1'
+        },
+        manualRowMove: true,
+        manualColumnMove: true,
+      });
+
+      getPlugin('manualRowMove').moveRows([4, 3, 2, 1, 0], 0);
+      getPlugin('manualColumnMove').moveColumns([4, 3, 2, 1, 0], 0);
```

Comment:
It was done intentionally. Currently, the code base on handling hooks performed by `ManualRowMove`, `ManualColumnMove`, `ManualColumnFreeze`. Using index mapper isn't handled by the Formulas plugin.

Another solution would assume adding extra hook to `IndexMapper` for moving. However, this brings up more questions:

1. What it should be called (so that it doesn't conflict with currently existing hooks)?
2. Will it be intuitive that there are two hooks (one from `IndexMapper`, another one from `ManualRowMove`/`ManualColumnMove` plugins) for moving rows/columns?
3. Removing hooks from `ManualRowMove`/`ManualColumnMove` plugins probably isn't a solution as hooks from the plugins respect idea to block some actions when `before~` hook is returning false, it counts `dropIndex` etc.

Reply:
Ok, thanks. So If I understand correctly, the _Formulas_ plugin works only with _ManualRowMove_/_ManualColumnMove_ and other plugins maintained by us and not with custom user implementation that uses the IndexMapper API eg. `columnIndexMapper.setIndexesSequence()` or `columnIndexMapper.moveIndexes()`?

Reply:
I've checked that and indeed... the _Formulas_ won't work when I play with `setIndexesSequence` or `moveIndexes`.

Reply:
To be clear, manipulating using IndexMapper's `setIndexesSequence` could work, but moving already not.

Reply:
It's worth adding that information to docs.